package.json

```diff
@@ -22,6 +22,6 @@
   "homepage": "https://github.com/arccoza/metalsmith-with-postcss",
   "dependencies": {
     "minimatch": "^3.0.0",
-    "postcss": "^5.0.19"
+    "postcss": "^6.0.9"
```

Comment:
A broader version range, `6.x`, so the plugin postcss dependency remains up to date.